S4TF_Tutorial_3.ipynb

```diff
@@ -0,0 +1,1772 @@
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "KaGgOzPQbGj3",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "import TensorFlow\n",
+        "import Foundation\n",
+        "import Python\n",
+        "\n",
+        "%include \"EnableIPythonDisplay.swift\"\n",
+        "IPythonDisplay.shell.enable_matplotlib(\"inline\")\n",
+        "let plt = Python.import(\"matplotlib.pyplot\")\n",
+        "let os = Python.import(\"os\")\n",
+        "let np = Python.import(\"numpy\")  // Make numpy available using np."
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "wwJGmDrQ0EoB"
+      },
+      "source": [
+        "This guide uses the [Fashion MNIST](https://github.com/zalandoresearch/fashion-mnist) dataset, which contains 70,000 grayscale images in 10 categories. The images show individual articles of clothing at low resolution (28 $\\times$ 28 pixels), as seen here:\n",
+        "\n",
+        "<table>\n",
+        "  <tr><td>\n",
+        "    <img src=\"https://tensorflow.org/images/fashion-mnist-sprite.png\"\n",
+        "         alt=\"Fashion MNIST sprite\"  width=\"600\">\n",
+        "  </td></tr>\n",
+        "  <tr><td align=\"center\">\n",
+        "    <b>Figure 1.</b> <a href=\"https://github.com/zalandoresearch/fashion-mnist\">Fashion-MNIST samples</a> (by Zalando, MIT License).<br/>&nbsp;\n",
+        "  </td></tr>\n",
+        "</table>\n",
+        "\n",
+        "Fashion MNIST is intended as a drop-in replacement for the classic [MNIST](http://yann.lecun.com/exdb/mnist/) dataset—often used as the \"Hello, World\" of machine learning programs for computer vision. The MNIST dataset contains images of handwritten digits (0, 1, 2, etc) in an identical format to the articles of clothing we'll use here.\n",
+        "\n",
+        "This guide uses Fashion MNIST for variety, and because it's a slightly more challenging problem than regular MNIST. Both datasets are relatively small and are used to verify that an algorithm works as expected. They're good starting points to test and debug code. \n",
+        "\n",
+        "We will use 60,000 images to train the network and 10,000 images to evaluate how accurately the network learned to classify images."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "Q_XrcNGNbejX",
+        "colab_type": "code",
+        "outputId": "c6983a16-952f-4916-825b-3c10cd39f603",
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 1465
+        }
+      },
+      "source": [
+        "let fileBaseCommand = \"https://raw.githubusercontent.com/tensorflow/swift-models/stable/MNIST/\"\n",
```

Comment:
I know you said that you'd probably rework the file loading, but this line defines a variable that doesn't seem to be used anywhere, and can probably be removed.